src/utils/test/clm_time_manager_test/test_clm_time_manager.pf

```diff
@@ -220,4 +221,156 @@ contains
 
   end subroutine check_DA_nstep
 
+  @Test
+  subroutine check_local_time(this)
+    class(TestTimeManager), intent(inout) :: this
+
+    integer :: secs
+    real(r8) :: londeg
+    integer  :: expected
+
+    ! Check for local noon at Greenich
+    londeg = 0.0_r8
+    secs = 3600*12
+    call set_date(yr=2, mon=3, day=1, tod=secs)
+    expected = secs
+    @assertEqual( expected, get_local_time( londeg ) )
+    londeg = 360.0_r8
+    @assertEqual( expected, get_local_time( londeg ) )
+
+    ! Check for local noon on otherside of the world
```

Comment:
For this and some of the other tests you added: I find it significantly easier to understand what tests are doing if they follow the generally-recommended unit testing organization of arrange-act-assert (or equivalently: setup-exercise-verify). That is, each test should test one thing. It gets hard to follow tests when they call the system under test multiple times in a single test.

Reply:
@billsacks and I talked about this. The general advice is to have one test per thing tested to make it easiest to read. It also helps in the theoretical sense so that broken tests will all be shown rather than just the first assert that fails within a test. Bill didn't think this had to change for this PR, but we do want to try to follow best practices for unit testing in general.